catppuccin-steam.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash 
+
+# Install dependencies
+if [ -f /usr/bin/pacman ] 
+then
+  sudo pacman -S curl wget unzip git
+elif [ -f /usr/bin/emerge ]
+then
+  sudo emerge -av curl wget unzip git
+elif [ -f /usr/bin/apt ]
+then
+   sudo apt install curl wget unzip git
+elif [ -d /nix ]
+then
+  sudo nix-shell -p curl wget unzip git
+elif [ -f /usr/bin/dnf ]
+then
+  sudo dnf install curl wget unzip git
+fi
+
+# Make temp directory
+cd $(mktemp -d)
+
+# Get the latest release of SFP and unzip it
+mkdir SFP 
+cd SFP 
+curl -s https://api.github.com/repos/PhantomGamers/SFP/releases/latest \
+| grep "SFP_UI-SelfContained-linux-x64.zip" \
+| cut -d : -f 2,3 \
+| tr -d \" \
+| wget -qi -
+unzip *.zip
+chmod +x SFP_UI
+./SFP_UI 
+
+# Install metro
+if [ -d ~/.steam ]
+then
+  mkdir -p ~/.steam/steam/skins/
+  git clone https://github.com/minischetti/metro-for-steam ~/.steam/steam/skins/catppuccin
+  export METRO="~/.steam/steam/skins/catppuccin"
+elif [ -d ~/.var/app/com.valvesoftware.Steam ]
+then
+  mkdir -p ~/.var/app/com.valvesoftware.Steam/.steam/steam/skins
+  git clone https://github.com/minischetti/metro-for-steam ~/.var/app/com.valvesoftware.Steam/.steam/steam/skins/catppuccin
+  export METRO="~/.var/app/com.valvesoftware.Steam/.steam/steam/skins/catppuccin"
+fin 
+ 
+# Install the metro patch
+cd ../ 
+mkdir UPMetro
+cd UPMetro
+git clone https://github.com/redsigma/UPMetroSkin
+cp -r 'Unofficial 4.x patch'/'Main Files'/* $METRO
+cd ../ 
+
+# Finally install the catppuccin flavor of the users choosing
+git clone https://github.com/catppuccin/steam.git
+cd steam
+printf "Which flavor would you like to install?\n 1) mocha\n 2) frappe\n 3) latte\n 4) macchiato"  
+read -r -p "--> " flavor
```

Comment:
`select` is probably better served for this [Bash Reference Manual#Select](https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#index-select)

Reply:
```select```
```bash: Unknown command: select```
Ran on a fresh install of Arch Linux

Reply:
![Writing a bash script with `set -euo pipefail` and a `select` statement in an ArchISO environment](https://user-images.githubusercontent.com/46626618/224579569-5d7e7813-3291-4970-b039-f73f7f2e5c3b.png)
I can't reproduce this even on Arch Linux

Reply:
+1 for this as well
currently there isn't a hint to chose the number or word, while also having the entire script fail when a number is chosen. this could be solved trivially by using this instead
```bash
flavors=("mocha" "frappe" "latte" "macchiato")

echo "please select a flavor:"
select f in "${flavors[@]}";
do
  [ -n "$f" ] && flavor="$t" && break;
  echo "invalid option. please select a number from the list"
done
```

Reply:
+1 for this as well
currently there isn't a hint to chose the number or word, while also having the entire script fail when a number is chosen. this could be solved trivially by using this instead
```bash
flavors=("mocha" "frappe" "latte" "macchiato")

echo "please select a flavor:"
select f in "${flavors[@]}";
do
  [ -n "$f" ] && flavor="$t" && break;
  echo "invalid option. please select a number from the list"
done
```